### PR TITLE
Resolving Issue #2541

### PR DESCRIPTION
**Pull request by sysres-dev**

### CODE REVIEW

File: lib/html.php [hunk 1]
```diff
@@ -2425,6 +2425,9 @@ function html_common_header($title, $selectedTheme = '') {
 		print get_md5_include_js($path2colorpicker);
 	}
 
+        if (file_exists('include/themes/custom.css')) {
+	        print get_md5_include_css('include/themes/custom.css');
```
Comment (netniV) [inline lib/html.php]: The indentation here is off.  Have you used spaces instead of tabs?

Comment (sysres-dev) [reply lib/html.php]: Yes that was my mistake. I used to work with spaces in my project. Sorry. That should be good now.

Comment (netniV) [reply lib/html.php]: I've pushed through an update to your repo to correct up the comment formatting.  That looked to be off anyway so I didn't think it fair to make you trudge through it.

**Comment by netniV** [comment]
I've approved the changes, just waiting on @cigamit to have a quick review to see whether he has any issues with adding this in.

**Pull request merged**